samples/nrf_rpc/protocols_serialization/server/sysbuild.conf

```diff
@@ -0,0 +1,4 @@
+SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
If you want to disable Partition Manager for MCUBoot, it's probably less confusing to do it for all configurations (regardless if you use MCUBoot or not) and make a partition map that includes all possible partitions (e.g. log history buffer is missing now). Otherwise, some combinations may not work properly in the sample.